sci-physics/yoda/yoda-1.9.5-r1.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
```

Comment:
Does it work with `python3_11`?

Reply:
I got QA warnings since there is no `python_single_target_python3_11` root yet.